### ISSUE TITLE
Autobuild interval from last key stroke

### ISSUE THREAD

**Issue by langit** [James-Yu/LaTeX-Workshop#4403]
> _Please fill the following fields with a star (*) and provide as much related information as possible._

## Pre-checks*
> _Please change the following `[ ]` to `[x]` for confirmation._
- [x] The feature request has not been suggested in this repository.

## The Missed*
> _Is your feature request related to a problem? Please provide a clear and concise description of what the problem is._

- the autobuild is quite annoying when you are editing the file and it will likely report errors. Therefore it would very nice to have a time interval on key strokes such that autobuild starts only after a certain amount of time from last key stroke.

## The Solution*
> _Please provide a solution you would like to have._

- autobuild starts only after a certain amount of time from last key stroke.

**Comment by James-Yu** [comment]
Auto-build won't be triggered by key stroke. Check your other settings.
